src/server/utils/pollModuleMap.js

```diff
@@ -126,7 +126,7 @@ async function pollModuleMap() {
   recordPollingForMonitor();
   startPollingMonitorIfNotAlready();
   try {
-    console.log('pollModuleMap: polling...');
+    console.info('pollModuleMap: polling...');
```

Comment:
lots of log level changes
how does this affect the log entries in production? are there some side effects we and users should be aware of? are they breaking? (e.g. searching through logs)

Reply:
Info is the default log level in production